consensus/src/consensus/remote_event_dispatcher.rs

```diff
@@ -337,7 +337,7 @@ impl<N: Network> Future for RemoteEventDispatcher<N> {
                     // Remove the peer from internal data structures.
                     self.state.write().remove_peer(&peer_id);
                 }
-                Ok(NetworkEvent::PeerJoined(_peer_id, _)) => {}
+                Ok(_) => {}
```

Comment:
I think adding information is better than removing it. Very much personal preference though.

```suggestion
                Ok(NetworkEvent::PeerJoined(_peer_id, _peer_info)) => {}
                Ok(NetworkEvent::DhtBootstrapped => {}
```

Reply:
I think for a "listen for events" it makes sense to ignore all that you don't want to listen for. Otherwise you'll need to update each location when you add another event.